backend/backend-infrastructure/ClassLibrary2/UnitOfWork.cs

```diff
@@ -0,0 +1,33 @@
+﻿using red_team.backend_infrastructure.Repositories;
+using red_team.Repositories.Interfaces;
+using System;
+
+namespace red_team.Repositories
+{
+    public class UnitOfWork : IDisposable
+    {
+        private SurveyContext db = new SurveyContext();
+        private SurveyRepository repository;
+ 
+        public SurveyRepository Surveys
+        {
+            get
+            {
+                if (repository == null)
+                    repository = new SurveyRepository(db);
```

Comment:
You should not use `new` Operator, Use DI. I already pointed you to the article how it should look like:
https://lostechies.com/derekgreer/2015/11/01/survey-of-entity-framework-unit-of-work-patterns/
(Injected Unit of Work Factory)